tests/freqai/test_freqai_interface.py

```diff
@@ -312,6 +320,8 @@ def test_follow_mode(mocker, freqai_conf):
     freqai.dd.load_all_pair_histories(timerange, freqai.dk)
 
     df = strategy.dp.get_pair_dataframe('ADA/BTC', '5m')
+    for i in range(5):
```

Comment:
I don't think it is necessary to add this to all the functions - only `start_backtesting()` and `extract_data_and_train()` should be sufficient.

Reply:
Kept only the check in `start_backtesting()` as there's no direct way of modifying the dataframe in the other test.